src/core/index.js

```diff
@@ -536,8 +557,8 @@ export function loginErrorMessage(m, error, type) {
     return i18n.html(m, ['error', 'login', 'lock.network']);
   }
 
-  // Custom rule error (except blocked_user)
-  if (error.code === 'rule_error') {
+  // Custom rule or hook error (except blocked_user)
+  if (error.code === 'rule_error' || error.code === 'hook_error') {
```

Comment:
```js
hooks: {
    loggingIn: function(context, done) {
        throw { code: 'hook_error', description: 'can you put <i>HTML</i> in here?' };
    },
```

I wonder if this is a risk, given that a `hook_error` is more likely to have reflected user input than a `rule_error`

Reply:
It's a good point. Currently you could put HTML in there, but for that to be reflected from user input the developer would have to specifically get it from the HTML within their hook, we don't give you those values in the context.

However, in the future there may be a case for making those available where HTML could inadvertently be written out, so we could clamp that down now and encode HTML when the error is output.

Reply:
I can see a use case for developer's wanting to display links in here. Seeing as the developer has to go out of their way to reflect user input in this field, let's cover it with guidance and make people aware that this will display HTML if you give it HTML.

Covered in https://github.com/auth0/lock/pull/1976/commits/ba9950aa65d7f3ceb30103e4a2c7fdfb6260d406